js_modules/dagster-ui/packages/ui-core/src/runs/useRunsForTimeline.tsx

```diff
@@ -443,7 +448,7 @@ export const useRunsForTimeline = ({
 
   const initialLoading =
     (loadingOngoingRunsData && !ongoingRunsData) ||
-    (loadingCompletedRunsData && !completedRunsQueryData) ||
+    (loadingCompletedRunsData && !completedRunsData) ||
```

Comment:
I think that this was a bug - since the querydata is present with `{data: undefined, loading: true}`, the completed runs data wasn't incorporated into the loading state really

Reply:
oops good catch